### PR TITLE
Changes to support Lock 11, Lock + Passwordless, and Management API 

### PR DESCRIPTION
**Pull request by aaguiarz**

### CODE REVIEW

File: RegularWebApp/views/index.jade [hunk 1]
```diff
@@ -28,7 +27,7 @@ block script
           redirectUrl: '#{env.AUTH0_CALLBACK_URL}',
           responseType: 'code',
           params: {
-            scope: 'openid'
+            scope: 'openid profile email'
```
Comment (luisrudge) [inline RegularWebApp/views/index.jade:30]: this is the default, so maybe there's no need?

File: RegularWebApp/views/index.jade [hunk 2]
```diff
@@ -37,11 +36,34 @@ block script
...
     function loginEmailCode() {
-      var lock = new Auth0LockPasswordless('#{env.AUTH0_CLIENT_ID}', '#{env.AUTH0_DOMAIN}');
-      lock.emailcode( {callbackURL: '#{env.AUTH0_CALLBACK_URL}'} );
+      var opts =  {
+        allowedConnections: ['email'],
+        auth: {
+          redirectUrl: '#{env.AUTH0_CALLBACK_URL}',
+          responseType: 'code',
+          params: {
+            scope: 'openid profile email'
```
Comment (luisrudge) [inline RegularWebApp/views/index.jade:45]: same

File: RegularWebApp/views/index.jade [hunk 3]
```diff
@@ -37,11 +36,34 @@ block script
...
-      var lock = new Auth0LockPasswordless('#{env.AUTH0_CLIENT_ID}', '#{env.AUTH0_DOMAIN}');
-      lock.magiclink( {callbackURL: '#{env.AUTH0_CALLBACK_URL}'} );
+      var opts =  {
+        allowedConnections: ['email'],
+        passwordlessMethod: 'link',
+        auth: {
+          redirectUrl: '#{env.AUTH0_CALLBACK_URL}',
+          responseType: 'code',
+          params: {
+            scope: 'openid profile email'
```
Comment (luisrudge) [inline RegularWebApp/views/index.jade:62]: same

**Review by luisrudge — CHANGES_REQUESTED** [review]

File: RegularWebApp/views/index.jade (hunk 3 above)
Comment (aaguiarz) [reply RegularWebApp/views/index.jade:62]: I have mixed feelings about it, I think it's better in samples to be explicit that we are actually specifying a scope.

**Review by luisrudge — APPROVED** [review]

File: RegularWebApp/views/index.jade (hunk 3 above)
Comment (luisrudge) [reply RegularWebApp/views/index.jade:62]: :shipit:

**Pull request merged**